classes/class-woothemes-sensei-certificates.php

```diff
@@ -181,6 +189,49 @@ private static function load_files() {
 		require_once dirname( __FILE__ ) . '/class-woothemes-sensei-certificates-data-store.php';
 	}
 
+	/**
+	 * Load background jobs.
+	 */
+	private static function load_background_jobs() {
+		require_once __DIR__ . '/background-jobs/class-sensei-certificates-create-certificates.php';
+
+		add_action( Sensei_Certificates_Create_Certificates::NAME, [ __CLASS__, 'run_create_certificates_job' ] );
```

Comment:
I was avoiding creating a new object on each request, but that's probably pretty minor performance wise since it is a small object with nothing going on in its constructor. I might keep it like this for now, though, considering this is also how we've done this in Sensei.